### ISSUE TITLE
Unable to find channel

### ISSUE THREAD

**Issue by MrFums** [peakxl/twitch-drop-claimer#119]
Using the docker compose image - getting the below error
![image](https://github.com/peakxl/twitch-drop-claimer/assets/57535680/92f8119f-8338-4ddd-add9-cbf43c90f546)

Tried with 
```

EA Sports FC 24
Lost Ark
Tom Clancy's Rainbow Six Siege
```

All of them seem to get the same error even though there are campaigns for each for each of them. 

Compose:

```
version: '3.5'
services:
  twitch-drop-claimer:
    image: peakxy/twitch-watcher:latest
    restart: unless-stopped
    environment:
      - auth_token=redacted
      - category=EA Sports FC 24
      - watchAlwaysTopStreamer=true
      - claimDrops=true
      #- channelsWithPriority=illojuan,alexby11,aroyitt
      #- noChannelFoundWait=5
      #- userAgent=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.108 Safari/537.36
      # - minWatching=15
      #- maxWatching=30
      #- streamerListRefresh=2
      #- streamerListRefreshUnit=hour
      #- browserScreenshot=false
      #- proxy=ip:port
      #- proxyAuth=username:password
    # volumes:
     # - "./screenshots:/usr/src/app/screenshots"
```

**Comment by QuentinJ54** [comment]
Hello.
You must indicate the category located in the url (example: https://www.twitch.tv/directory/category/ea-sports-fc-24) and therefore must put "ea-sports-fc-24".

**Comment by MrFums** [comment]
Thanks for the reply, seems to still not be working with the eac-sports-fc-24 or lost-ark

![image](https://github.com/peakxl/twitch-drop-claimer/assets/57535680/f2e9ee91-5ed6-4769-baaf-8ad431540802)



`- category=lost-ark`

**Comment by QuentinJ54** [comment]
hello,

Didn't you leave a space for something we can't see?

**Comment by peakxl** [comment]
@Stiphr Please make sure you pull the newest image with "docker compose pull" before "docker compose up".

**Comment by peakxl** [comment]
@Stiphr I still cannot replicate this, even on Windows 11 with docker desktop. I think this has to be related with your twitch account settings.

**Comment by peakxl** [comment]
Fixed by bc065ed092eed6ec7d98a4774b01b1502ae2910e
Please open a new issue for this type of issues.
